runbotics/runbotics-desktop/src/action/microsoft/excel/excel.types.ts

```diff
@@ -158,8 +158,8 @@ export interface WorkbookCell {
     text?: string;
     numberFormat?: string;
     address?: string;
-    formula?: string;
-    valueType?: string;
+    formula?: Array<string>;
+    valueType?: Array<string>;
```

Comment:
I would stick to the convention `string[]`

Reply:
k, changes all arrays in the file :D